uber/models.py

```diff
@@ -806,10 +808,6 @@ def _cost_and_leader(self):
             self.leader.ribbon = c.DEALER_RIBBON
 
     @property
-    def is_new(self):
-        return not instance_state(self).persistent
-
-    @property
```

Comment:
Unrelated to the above fix, but I noticed there was a copy/paste here.  This function is already defined in ``MagModel`` and there's no point to defining it twice, so I deleted the duplicate.